index.html

```diff
@@ -150,9 +150,13 @@
             margin: 0 auto;
         }
 
-        .downloadexplain {
+        .downloadarea {
             color: #fff;
-            font-size: 30px;
+        }
+
+        .linebrBefore:before {
```

Comment:
It should have been solved that way. I have corrected it to use margin instead now.